math/greatest_common_divisor.go

```diff
@@ -1,22 +1,15 @@
-"""
-Greatest Common Divisor.
-Wikipedia reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
-
-"""
-
-
 package main
 
 import ("fmt")
 
 func gcd(a, b int) int {
 	if b == 0 {
 		return a
-	} else {
-		return gcd(b, a%b)
-	}
+	} 
+   return gcd(b, a%b)
```

Comment:
It seems wrong indentation. Did you use golint or some lint tool to format it?

Reply:
Now I've fixed the indentation with the linter.